### PR TITLE
revert: remove Coin Wallet

### LINKED ISSUE #12970: Suggest a wallet (add back): Coin Wallet
### Wallet name

Coin Wallet

### What type of wallet?

EOA

### Wallet description

Coin Wallet is a non-custodial multicurrency wallet for multiple platforms.

### Wallet logo

yRDhG3sqmo-228

### Background color for brand logo

#68c481

### URL to the project

https://coin.space/

### When did the wallet go live to users?

2015

### What are the fees users may be paying for using your wallet product?

Network fee for Ethereum

### Does the wallet have an active development team?

yes https://github.com/CoinSpace/CoinSpace

### Is the wallet globally accessible?

no KYC, no limitation

### Is the wallet available in multiple languages?

```
languages_supported: [
      "id",
      "bs",
      "cs",
      "de",
      "en",
      "es",
      "fil",
      "fr",
      "hr",
      "it",
      "hu",
      "nl",
      "nb",
      "pl",
      "pt-br",
      "ro",
      "sr",
      "sv",
      "vi",
      "tr",
      "ru",
      "uk",
      "th",
      "km",
      "ko",
      "ja",
      "zh-cn"
    ],
```

### What social links are there for the project?

* https://twitter.com/CoinAppWallet
* https://www.facebook.com/coinappwallet/
* https://www.linkedin.com/company/coin-space/

### Does the wallet have a mobile app? If yes, which operating systems are supported (iOS, Android)?

https://apps.apple.com/app/apple-store/id980719434
https://play.google.com/store/apps/details?id=com.coinspace.app

### Does the wallet have a desktop app? If yes, which operating systems are supported (Windows, Mac, Linux)?

Yes, Windows, Mac, Linux

https://www.microsoft.com/store/apps/9NBLGGH5PXJQ
https://apps.apple.com/app/coin-wallet-bitcoin-crypto/id980719434?platform=mac
https://snapcraft.io/coin

### Does the wallet have a browser extension? If yes, which browsers are supported (Chromium, Firefox, Safari)?

No browser extension, just a full featured web app https://coin.space/wallet/

### Is it a hardware wallet?

No

### What is the availability of your source code?

Open source

### What license is the wallet software released under?

https://github.com/CoinSpace/CoinSpace/blob/master/LICENSE

### Who holds the private keys?

User, because it is non-custodial wallet

### Please describe the measures taken to ensure the wallet's security and provide documentation wherever possible

Because it is live and open source from 2015 https://github.com/CoinSpace/CoinSpace

### Has the wallet's smart contract code or security modules been audited?

-

### Does the wallet have an internal security team?

yes

### Any other security testing that should be noted?

Wallet supports 2FA using any FIDO-certified security key.

https://hackerone.com/coinspace

### Scam protection?

Wallet uses audited crypto list https://github.com/CoinSpace/crypto-db

### Does the wallet support connecting to Ethereum applications?

Not yet, but planned

### Does the wallet support connecting to a hardware wallet?

No

### Does the wallet support importing Ethereum RPC endpoints?

no

### Does the wallet support viewing and interacting with NFTs?

No

### Does the wallet support staking directly?

no

### Does the wallet support swaps directly?

yes

### Does the wallet support bridging directly?

no

### Does the wallet support multi-chain networks?

yes

### Does the wallet support Ethereum layer 2 networks?

yes

### Does the wallet allow the user to customize gas fees?

yes

### Does the wallet support sending transactions to ENS addresses?

yes

### Does the wallet support importing or automatically querying and displaying ERC-20 tokens?

yes

### Does the wallet support EIP-1559 (type 2) transactions?

yes

### Does the wallet have any recovery options?

Your Keys, Your Crypto!

### Does the wallet have fiat on-ramps through credit/debit cards, wire transfers, or bank transfers (ACH)?

yes https://coin.space/buy-ethereum/

### Does the wallet support withdrawals to fiat?

yes

### Is the wallet a multi-signature wallet?

no

### Does the wallet support social recovery?

no, Your Keys, Your Crypto!

### Does the wallet support bundled transactions?

no

### Who can the ethereum.org team can contact regarding the wallet in future?

support@coin.space and https://t.me/nverkhovin

### Does the wallet have a dedicated support team?

Yes, we have support agents. 1 business day.
https://support.coin.space/hc/en-us
[support@coin.space](mailto:support@coin.space)

### What educational resources/documentation do you provide to users?

Help Center https://support.coin.space/hc/en-us

### Does the wallet have any integrated tools not mentioned above?

Coin Wallet supports second factor (2FA) using FIDO-certified security key (hardware) to complete any sensitive actions.

### Would you like to work on this issue?

- [X] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by mahnunchik**
Add Coin Wallet back.

## Description

It seems Coin Wallet was removed by mistake. Coin Wallet supports EIP-1559.

I've updated features list as well.

## Related Issue
- Closes #12970
* https://github.com/ethereum/ethereum-org-website/issues/6772
* https://github.com/ethereum/ethereum-org-website/pull/12308
* https://github.com/ethereum/ethereum-org-website/pull/12308/commits/62a642b44ca4bdab4ce13efd7cd918b6e5036ad5

### CODE REVIEW

**Comment by corwintines** [comment]
Thanks for bringing this up @mahnunchik

Since this wallet was out of date, I think we should make sure the information we have is updated. Do you mind filling this out so we can have an up to date listing for Coin Wallet and review it in its current state.

**Comment by mahnunchik** [comment]
@corwintines done https://github.com/ethereum/ethereum-org-website/issues/12970

File: src/data/wallets/wallet-data.ts [hunk 1]
```diff
@@ -408,6 +409,76 @@ export const walletsData: WalletData[] = [
     onboard_documentation: "https://docs.safe.global/",
     documentation: "https://docs.safe.global/",
   },
+  {
+    last_updated: "May 10, 2024",
+    name: "Coin wallet",
```
Comment (wackerow) [inline src/data/wallets/wallet-data.ts:414]: Per usage on https://coin.space/

```suggestion
    name: "Coin Wallet",
```

File: src/data/wallets/wallet-data.ts [hunk 2]
```diff
@@ -408,6 +409,76 @@ export const walletsData: WalletData[] = [
...
+      "sv",
+      "vi",
+      "tr",
+      "ru",
+      "uk",
+      "th",
+      "km",
+      "ko",
+      "ja",
+      "zh-cn"
```
Comment (wackerow) [inline src/data/wallets/wallet-data.ts:446]:
```suggestion
      "zh"
```

Comment (wackerow) [reply src/data/wallets/wallet-data.ts:446]: Never mind, I see this parses just fine as is.

**Review by wackerow — APPROVED** [review]
Lgtm! Thanks @mahnunchik... I can't commit to your branch so I'm just going to bring this in and we can patch the capitalization of the name separately.

**Comment by wackerow** [comment]
Actually, looks like there are some conflicts which I'm unable to clear myself. @mahnunchik Mind fixing those up and then we can merge in?

**Comment by mahnunchik** [comment]
@wackerow I've rebased on top of current dev branch.

**Review by wackerow — APPROVED** [review]
Lgtm! Thanks @mahnunchik

**Comment by wackerow** [comment]
@all-contributors please add @mahnunchik for tool

**Pull request merged**